src/components/HowItWorksDialog.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <base-dialog
+    id="how-it-works"
+    v-model="show"
+    title="Looking To Play?"
+    :opacity="1"
+    data-cy="create-game-dialog"
+  >
+    <template #activator>
+      <v-btn
+        color="surface-2"
+        class="text-subtitle-1 px-10"
+        variant="text"
+        size="x-large"
+        data-cy="how-it-works-button"
+      >
+        How it works
+      </v-btn>
+    </template>
+    <template #body>
+      <section class="d-flex flex-column">
+        <span class="my-2">Our weekly play sessions are open to everyone:</span>
+        <li>Wednesday nights at 8:30pm EST</li>
+        <li>Thursdays at 12pm EST</li>
+        <span class="my-2"> Can&#39;t find a match? Join us on 
+          <a 
+            class="text-anchor text-decoration-none" 
+            href="https://discord.gg/9vrAZ8xGyh" 
+            target="_blank"
+          >discord 
+          </a> to chat with other players and to see who is available to play at any time.</span>
```

Comment:
Format the newlines so new tags start new lines
```suggestion
          <a 
            class="text-anchor text-decoration-none" 
            href="https://discord.gg/9vrAZ8xGyh" 
            target="_blank"
          >
            discord 
          </a>
          to chat with other players and to see who is available to play at any time.
        </span>
```